en/controllers/request-response.rst

```diff
@@ -181,17 +181,17 @@ subdirectory. The attributes you can use are::
     // Assume the current request URL is /subdir/articles/edit/1?page=1
 
     // Holds /subdir/articles/edit/1?page=1
-    $request->here();
+    $here = $request->here();
 
     // Holds /subdir
-    $request->getAttribute('base');
+    $base = $request->getAttribute('base');
```

Comment:
base and webroot are only different by a `/` unless URL rewriting is disabled, then they are very different.